package.json

```diff
@@ -17,7 +17,8 @@
     "stop:all": "npm stop; npm run stop:mockapi",
     "chimp": "NODE_ENV=acceptance chimp --mocha --browser=htmlunit --path=./test/acceptance/spec/",
     "chimp:dev": "NODE_ENV=acceptance chimp --mocha --path=./test/acceptance/ --watch --browser=firefox",
-    "lint": "eslint .",
+    "lint": "node_modules/.bin/eslint $FIXLINT .",
```

Comment:
Are you sure we need `node_modules/.bin/`?
I think npm does some magic to add that to the path.